.github/workflows/format.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install JuliaFormatter and format
         run: |
-          julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter", version="0.22.10"))'
+          julia  -e 'using Pkg; Pkg.add(PackageSpec(name="JuliaFormatter", version="1.0.9"))'
```

Comment:
this is a side note from this pr:

How about adapting a `JuliaFormatter` style for this project?
This works by placing a `.JuliaFormatter.toml` file in the main dir with e.g. content:
```toml
# Configuration file for JuliaFormatter.jl
# For more information, see: https://domluna.github.io/JuliaFormatter.jl/stable/config/

always_for_in = true
always_use_return = true
margin = 80
remove_extra_newlines = true
short_to_long_function_def = true
```

Reply:
Adding a style could maybe make sense. But would it not mainly be if we want to deviate from the default values? I haven't thought so much about the actual style that is used actually.